SU-SWS/jumpstart_ui
# D8 core 971 gutter

# READY FOR REVIEW


# Summary
Adjust the space between the aside and the main content on Stanford pages 

# Needed By (Date)
- RC1

# Urgency
- Nice to have?

# Steps to Test
- Checkout this branch
- Verify that for breakpoints xl and larger, there's an extra 150px between the main content and the side bar.

## Smaller than xl
![Screen Shot 2019-12-20 at 7 58 25 PM](https://user-images.githubusercontent.com/284440/71302786-551eb900-2364-11ea-8c51-f54d8f3c9370.png)

## XL
![Screen Shot 2019-12-20 at 7 58 58 PM](https://user-images.githubusercontent.com/284440/71302791-6a93e300-2364-11ea-982c-c7d589bf8b98.png)

## XXL
![Screen Shot 2019-12-20 at 7 59 19 PM](https://user-images.githubusercontent.com/284440/71302800-7ed7e000-2364-11ea-9364-9de3b895f188.png)



# Affected Projects or Products
D8Core

# Associated Issues and/or People
- JIRA ticket
https://stanfordits.atlassian.net/browse/D8CORE-971

- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
